desk-controller/controllers/http_controller.py

```
@@ -0,0 +1,42 @@
+import os
+from dotenv import load_dotenv
+import logging
+
+
+class HttpController:
+    def __init__(self) -> None:
+        load_dotenv(dotenv_path="../.env")
+        self.BASE_URL = os.getenv("API_BASE_URL")
+        self.logger = logging.getLogger(__name__)
+
+    def send_request(self, path, type="GET", headers={}, payload={}):
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)